<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloader</class>
 <widget class="QMainWindow" name="downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1121</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>1121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="close">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>10</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>close</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(193, 0, 0);
border-radius:8px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Downloader</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>55</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Pictures/icons8-download-96.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Pictures/icons8-youtube-512.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="min">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>10</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>minimize</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 229, 0);
border-radius:8px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="max">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>10</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>normal</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
border-radius:8px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>191</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway SemiBold</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);
border-radius:0px;</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway SemiBold</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);
border-radius:0px;</string>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway SemiBold</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);
border-radius:0px;</string>
      </property>
      <property name="text">
       <string>About Me</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>710</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>680</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>650</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>620</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>590</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>761</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 232, 232);
border-radius:15px;</string>
     </property>
     <property name="placeholderText">
      <string>type the Link</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
border-radius:15px;</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>290</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 232, 232);
border-radius:15px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>761</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 232, 232);
border-radius:15px;</string>
     </property>
     <property name="placeholderText">
      <string>save to...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>520</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Pictures/Fast loading-amico.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>downloader</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1093</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>downloader</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1013</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>downloader</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1053</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
